src/main/kotlin/no/nav/modiapersonoversikt/storage/LocalStorageProvider.kt

```diff
@@ -1,26 +1,43 @@
 package no.nav.modiapersonoversikt.storage
 
-import no.nav.modiapersonoversikt.model.Melding
+import no.nav.modiapersonoversikt.model.Notifikasjon
+import no.nav.modiapersonoversikt.model.NotifikasjonDTOIn
+import no.nav.modiapersonoversikt.model.NotifikasjonDTOOut
 import java.time.LocalDateTime
 import java.util.*
 
 class LocalStorageProvider : StorageProvider {
-
-    override fun getMeldinger(): List<Melding> {
-        return emptyList()
+    private var meldinger: MutableMap<UUID, Notifikasjon> = mutableMapOf()
+    private var sistLest: MutableMap<String, LocalDateTime> = mutableMapOf()
+
+    override fun hentNotifikasjoner(ident: String): List<NotifikasjonDTOOut> {
+        val sistLest = hentSistLest(ident)
+        return meldinger
+                .entries
+                .map { NotifikasjonDTOOut(it.key, it.value) }
+                .map { it.copy(settTidligere = it.opprettetDato.isBefore(sistLest)) }
     }
 
-    override fun getMelding(id: UUID): Melding {
-        return Melding(UUID.randomUUID(), "Tittel", "TEST", "lenke", LocalDateTime.now())
+    override fun hentNotifikasjon(id: UUID): NotifikasjonDTOOut? {
+        return meldinger[id]
+                ?.let {  NotifikasjonDTOOut(id, it) }
```

Comment:
Denne skal vel ikke ha `?` siden denne alltid returnerer noe? i tillegg:
```
override fun hentNotifikasjon(id: UUID): NotifikasjonDTOOut = meldinger.getOrElse(id) { NotifikasjonDTOOut(id, it) }
```
:)

Reply:
Må nesten ha `?` her. For kan være tilfeller hvor det ikke finnes en notifikasjon med `id`

Reply:
Sorry, jeg leste denne feil... Ignore.